src/main/java/pl/wrapper/parking/infrastructure/inMemory/ParkingDataRepository.java

```diff
@@ -0,0 +1,37 @@
+package pl.wrapper.parking.infrastructure.inMemory;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.serializer.support.SerializationFailedException;
+import org.springframework.stereotype.Component;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.util.HashMap;
+
+@Component
+public class ParkingDataRepository extends InMemoryRepositoryImpl<Integer, SomeObject> { //K and V types can/should be changed, but leave that for those who have the other two tasks
```

Comment:
create another class like this to store ParkingEndpoint data. You only need to implement the constructor and init method